main.cpp

```diff
@@ -12,76 +12,202 @@ What you create in this project will be used as the basis of Project 5 in the co
     
-Sub Object 3: Environment Controls
-    Name 3 things you'll find on the:    Environment Controls
-        1)
-        2)
-        3)
-    Name 3 things you can do with the:   Environment Controls
-        1)
-        2)
-        3)
-
-Sub Object 4: Infotainment System
-    Name 3 things you'll find on the:    Infotainment System
-        1)
-        2)
-        3)
-    Name 3 things you can do with the:   Infotainment System
-        1)
-        2)
-        3)
-
-Sub Object 5: Seat 
-    Name 3 things you'll find on the:    Seat
-        1)
-        2)
-        3)
-    Name 2 things you can do with the:   Seat
-        1)
-        2)
+    The 'pixels' property must be represented with an array of Pixel instances, as the screen has more than 1 row of pixels.
+        Arrays have not been discussed and can't be used in this project.
+        Instead, we can use an Integer primitive to store the Number of Pixels:
+
+    Display:
+        Number of Pixels
+        Amount of Power consumed (milliwatt-hours)
+        Brightness
+        width in cm
+        height in cm
+************************
+
+1) Fill in #5 - #9 with plain-english UDTs for the 5 properties you created for UDT #10
+    example: 
+        If #10's first property was 'Engine', then your `Thing 5)` will be `Engine` and 
+        you will need to provide 5 properties and 3 member functions of that Engine object in plain English
 */
+
+/*
+Thing 1) Restaurant
+5 properties:
+    1) Number of tables
+    2) Number of employees
+    3) Restaurant name
+    4) Size of restaurant in square feet
+    5) Amount of profit each day
+3 things it can do:
+    1) Feed hungry customers
+    2) Charge customers
+    3) Provide salary to employees
+ */
+
+/*
+Thing 2) Family
+5 properties:
+    1) Size of family
+    2) Number of kids
+    3) Number of parents
+    4) Mother's name
+    5) Father's name
+3 things it can do:
+    1) Eat dinner together
+    2) Go to church
+    3) Bicker
+ */
+
+/*
+Thing 3) Synthesizer
+5 properties:
+    1) Number of keys
+    2) Number of synth voices
+    3) Name of synth
+    4) Output gain
+    5) Number of outputs
+3 things it can do:
+    1) Play sound
+    2) Change output gain amount
+    3) Change number of outputs
+ */
+
+/*
+Thing 4) Laptop
+5 properties:
+    1) Model year
+    2) Amount of RAM
+    3) Amount of storage
+    4) Screen size
+    5) Manufacturer
+3 things it can do:
+    1) Compile code
+    2) Browse the web
+    3) Play movies
+ */
+
+/*
+Thing 5) Compressor
+5 properties:
+    1) Hysteresis
+    2) Gain 
+    3) Threshold
+    4) Range
+    5) Ratio
+3 things it can do:
+    1) Limit 
+    2) Compress 
+    3) Expand 
+ */
+
+/*
+Thing 6) EQ 
+5 properties:
+    1) highBandQ
+    2) midBand1Q
+    3) midband2Q
+    4) lowbandQ
+    5) highBand gain
+3 things it can do:
+    1) low shelf shape EQ
+    2) high shelf shape EQ
+    3) peak shape EQ
```

Comment:
These are objects, not actions
- [ ] revise

Reply:
I was thinking more "create low shelf shaped EQ", but i changed them